src/wmtk/simplex/closed_star.cpp

```diff
@@ -0,0 +1,35 @@
+#include "closed_star.hpp"
+
+#include <wmtk/TetMesh.hpp>
+#include <wmtk/TriMesh.hpp>
+
+#include "coface_cells.hpp"
+#include "simplex_boundary.hpp"
+
+namespace wmtk::simplex {
+
+SimplexCollection closed_star(const Mesh& mesh, const Simplex& simplex, const bool sort_and_clean)
+{
+    SimplexCollection collection(mesh);
+
+    collection.add(simplex);
+
+    const SimplexCollection coface_cells_collection =
+        mesh.top_simplex_type() == PrimitiveType::Face
+            ? coface_cells(static_cast<const TriMesh&>(mesh), simplex, false)
```

Comment:
perhaps this should be a switch/case thing. soon we'll have edge/point meshes too

Reply:
sure, I created an issue for that